### DISCUSSION TITLE
How to scroll using keyboard in 'Custom patch' panel?

### DISCUSSION THREAD

**Discussion by pm-gmr** [jesseduffield/lazygit#3152] [Q&A]
When I'm creating a patch from an existing commit, I can't change the focus to "Custom patch" panel to scroll it up/down. The only way to scroll it is using mouse wheel. Is there a way to do that using keyboard? PgUp and PgDown only scroll the `Patch` panel i.e. the panel where I create the custom patch from.

It would be great if I could use `Tab` key to switch to that panel and scroll just like we do when staging/unstaging changes.

**Comment by mark2185** [comment]
Unfortunately that's not yet possible, [an issue](https://github.com/jesseduffield/lazygit/issues/1627#issue-1083019779) was created a while ago. Contributions are welcome, though!
